api/lib/application/security-pre-handlers.js

```diff
@@ -145,6 +146,28 @@ async function checkUserIsAdminInOrganizationOrHasRolePixMaster(request, h) {
   return _replyWithAuthorizationError(h);
 }
 
+async function checkUserBelongsToOrganizationManagingStudents(request, h) {
+  if (!request.auth.credentials || !request.auth.credentials.userId) {
```

Comment:
Petite suggestion d'amélioration avec lodash : 

```suggestion
  if (!_.has(request, 'auth.credentials.userId')) {
```